test/index.js

```diff
@@ -1 +1,5 @@
 require('./measures/measures-data-spec');
+require('./util/is-inverse-benchmark-spec');
+require('./util/format-decile-generator-spec');
+require('./util/format-csv-record-spec');
+require('./benchmarks/benchmarks-data-spec');
```

Comment:
Hmm should `test/benchmarks/benchmarks-data-spec.js` be removed? Doesn't seem to be anything there. Or if there is an intention to add tests maybe we want to add a file with a comment to that effect.